README.md

```diff
@@ -23,42 +23,28 @@ C++ Idiomatic Clients for [Google Cloud Platform][cloud-platform] services.
 This library supports the following Google Cloud Platform services with clients
 at an [Alpha](#versioning) quality level:
 
-- [Google Cloud Bigtable](bigtable)
+- [Google Cloud Bigtable](google/cloud/bigtable#google-cloud-bigtable)
 
 The libraries in this code base likely do not (yet) cover all the available
-APIs. See the [`googleapis` repo](https://github.com/googleapis/googleapis)
-for the full list of APIs callable using gRPC.
-
-## Quick Start
-
-To build the available libraries and run the tests, run the following commands
-after cloning this repo:
-
-```bash
-git submodule init
-git submodule update --init --recursive
-cmake -H. -Bbuild-output
-cmake --build build-output
-(cd build-output && ctest --output-on-failure)
-```
-
-On Linux and macOS you can speed up the build by replacing the
-`cmake --build build-output` step with:
-
-```bash
-cmake --build build-output -- -j $(nproc)
-```
-
-On Windows with MSVC use:
-
-```bash
-cmake --build build-output -- /m
-```
-
-Consult the `README.md` file for each library for links to the examples and
-tutorials.
-
-## Build Dependencies
+APIs.
+
+## Table of Contents
+
+- [Requirements](#requirements)
+  - [Compiler](#compiler)
+  - [Build Tools](#build-tools)
+  - [Libraries](#libraries)
+  - [Tests](#tests)
+- [Install Dependencies](#install-dependencies)
+  - [Ubuntu (Bionic Beaver)](#ubuntu-bionic-beaver)
+  - [Ubuntu (Trusty)](#ubuntu-trusty)
```

Comment:
Should we include instructions to other distributions?  GCE officially supports CentOS and SuSE, which use different packaging.  If the answer is yes, that does not need to happen in this PR, but maybe "other distros" and some hints from there?

Reply:
Are you referring to "other distros" such as what's inside `ci/`?

Reply:
Right. The current PR only covers a subset of the different distributions we decided to support, until we have instructions for all of the distros we support what should we say?